modules/deploy/partials/high-availability.adoc

```diff
@@ -371,6 +371,149 @@ ID HOST PORT RACK
 
 endif::[]
 
+== Multi-AZ deployments
+
+A multi-AZ deployment is one in which we deploy a single Redpanda cluster that has brokers distributed over multiple availability zones. The basis for this architecture is Redpanda’s rack awareness feature, in which each zone becomes a “rack”. This allows Redpanda to place replicas with respect to failure domains, resulting in a cluster that can survive a zone outage.
```

Comment:
Please see comment about not duplicating introductory info, and pls remove instances of "we".